client/src/components/Onboarding/OnboardingStatus.module.scss

```diff
@@ -3,9 +3,8 @@
   margin-right: 0.5em;
 }
 
-.loading-icon {
-  display: inherit;
-  width: inherit;
+.loading {
+  width: auto; /* overwrite <LoadingSpinner> */
 }
```

Comment:
The `display: inherit;` seemed unnecessary, but I open to hear the case for it.

The `width: auto` is a simpler, and equally reliable (here), solution that gives the same result.

<details><summary>About <code>width: inherit</code></summary>

> I am intrigued by the seemingly intuitive CSS values `inherit`, `initial`, and `unset`. But, most times I think I find a use case for `inherit` and `initial`, I end up choosing a different value.

The `width: inherit` works, but not why you think. It "inherits" the computed value of its parent⁰, which is computed based on the value of `auto`¹. Also, width is not an inherited property², so `width: inherit` "specifies a behavior that typically makes relatively little sense".³

> ⁰ See [CSS → Inherit | MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/inherit).
> ¹ The parent element does not have a set `width`, so its width defaults to `auto`. See "Initial value" at [CSS → Width | MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/width#formal_definition).
> ² See "Some properties do not inherit" at [CSS → Inheritance | MDN](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Cascade_and_inheritance#inheritance).
> ³ See [CSS → Inherit | MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/inherit).

</details>